react_components/Card.js

```diff
@@ -167,4 +193,4 @@ CardComponent.propTypes = {
   })
 };
 
-export default CardComponent;
+export default injectIntl(CardComponent);
```

Comment:
add code comment that this injects this.props.intl which is used above to get the formatMessage method